### DISCUSSION TITLE
请问这个DevClient的连接问题如何解决？

### DISCUSSION THREAD

**Discussion by Yukihana-Lamy** [the1812/Bilibili-Evolved#4682] [提问 / Q&A]
我在篡改猴里开启了Bilibili Evolved，访问了B站网页版，点击了DevClient的【连接】，但是连接失败，请问这个问题如何解决？如何使用DevClient功能？

![DevClient](https://github.com/the1812/Bilibili-Evolved/assets/103800975/01511b0a-b074-4f0e-85f0-7cc6a3e3dc64)

**Comment by the1812** [comment]
你都不知道是干啥用的就随便装?

**Comment by Tinhone** [comment]
DevClient 组件是开发用的啦，正常使用不需要装的
